memos/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      # <app-id>_<web-container-name>_1
+      # e.g. 'btc-rpc-explorer_web_1'
+      # Note that the '_1' at the end is needed
+      APP_HOST: memos_main_1
+      APP_PORT: 5230
+      PROXY_AUTH_WHITELIST: "/api/*"
+
+  main:
+    image: neosmemo/memos:0.21
```

Comment:
Let's lock this in with a specific multi-arch digest so all users get the same image (even across app restarts):

```
$ sudo docker pull neosmemo/memos:0.21
0.21: Pulling from neosmemo/memos...
Digest: sha256:1d295e66cc2e75f1645872fc4a372f8f58efad17902f432758762d747984d6fb
Status: Downloaded newer image for neosmemo/memos:0.21
docker.io/neosmemo/memos:0.21
```

Reply:
Well, it turns out I forgot about the digest in every submission. I've fixed it.